Subscribers/Article.php

```diff
@@ -230,10 +230,10 @@ public function extendBackendArticle(\Enlight_Event_EventArgs $args)
                     $this->modelManager->flush();
                 }
 
-                $sourceIds = $this->modelManager->getConnection()->fetchColumn(
+                $sourceIds = $this->modelManager->getConnection()->executeQuery(
                     'SELECT source_id FROM s_plugin_connect_items WHERE article_id = ?',
                     [$article->getId()]
-                );
+                )->fetchAll(\PDO::FETCH_COLUMN);
```

Comment:
:+1: 
This will fetch all sourceIds by given articleId and will work for variants.
I'm wondering only if the article has 4-5K variants. Maybe we need to check if the articles must be updated via CRON just update column cron_update.
@keulinho what do you think.